app/containers/EmojiPicker/ReactionPicker.js

```diff
@@ -0,0 +1,254 @@
+		const { searchQuery } = this.state;
+
+		if (!newSearchQuery) {
+			return this.setState({ searchResults: {}, searchQuery: '' });
+		}
+
+		if (searchQuery !== newSearchQuery) {
+			const searchResults = emojis.filter(
+				emojiName => emojiName.indexOf(newSearchQuery.toLowerCase()) === 0
+			);
+
+			this.setState({ searchQuery: newSearchQuery, searchResults });
+		}
+	};
+
+	updateFrequentlyUsed() {
+		const frequentlyUsed = map(this.frequentlyUsed.slice(), (item) => {
+			if (item.isCustom) {
+				return item;
+			}
+			return emojify(`${ item.content }`, { output: 'unicode' });
+		});
+		this.setState({ frequentlyUsed });
+	}
+
+	updateCustomEmojis() {
+		const customEmojis = map(this.customEmojis.slice(), item => ({
+			content: item.name,
+			extension: item.extension,
+			isCustom: true
+		}));
+		this.setState({ customEmojis });
+	}
+
+	renderCategory(category, i) {
+		const { frequentlyUsed, customEmojis } = this.state;
+		const { emojisPerRow, width, baseUrl } = this.props;
+		let emojiCategory;
+
+		if (i === 0) {
+			emojiCategory = frequentlyUsed;
+		} else if (i === 1) {
+			emojiCategory = customEmojis;
+		} else {
+			emojiCategory = emojisByCategory[category];
+		}
+		return (
+			<EmojiCategory
+				emojis={emojiCategory}
+				onEmojiSelected={emoji => this.onEmojiSelected(emoji)}
+				style={styles.categoryContainer}
+				size={emojisPerRow}
+				width={width}
+				baseUrl={baseUrl}
+			/>
+		);
+	}
+
+	renderItem(emoji, size) {
+		const { baseUrl, onEmojiSelected } = this.props;
+		return (
+			<TouchableOpacity
+				activeOpacity={0.7}
+				key={emoji.isCustom ? emoji.content : emoji}
+				onPress={() => onEmojiSelected(emoji)}
+				testID={`reaction-picker-${ emoji.isCustom ? emoji.content : emoji }`}
+			>
+				{renderEmoji(emoji, size, baseUrl)}
+			</TouchableOpacity>
+		);
+	}
+
+	renderCategories() {
+		const { show, searchQuery, searchResults } = this.state;
+		const { emojisPerRow, width, baseUrl } = this.props;
+
+		if (!show) {
+			return <ActivityIndicator style={styles.loader} />;
+		}
+
+		return (
+			<ScrollView {...scrollProps} style={styles.background}>
+				{(searchQuery
+					&& (searchResults.length > 0 ? (
+						<View style={styles.background} {...scrollProps}>
+							<EmojiCategory
+								emojis={searchResults}
+								onEmojiSelected={emoji => this.onEmojiSelected(emoji)}
+								style={styles.categoryContainer}
+								size={emojisPerRow}
+								width={width}
+								baseUrl={baseUrl}
+							/>
+						</View>
+					) : (
+						<Text style={styles.noEmojiFoundText}>{I18n.t('Emoji_Not_Found')}</Text>
+					)))
+					|| categories.tabs.map((tab, i) => (
+						<View key={tab.category} style={styles.background} {...scrollProps}>
+							<Text style={styles.categoryTitle}>{tab.title}</Text>
+							{this.renderCategory(tab.category, i)}
+						</View>
+					))}
+			</ScrollView>
+		);
+	}
+
+	render() {
+		return (
+			<View>
+				<RCTextInput
```

Comment:
Use <Searchbox /> instead

Reply:
What do you mean?

Reply:
`<Searchbox />` (Github don't show tags in plain text 🙄)